src/answers-headless.ts

```diff
@@ -165,21 +165,11 @@ export default class AnswersHeadless {
     return response;
   }
 
-  async executeUniversalAutoComplete(): Promise<AutocompleteResponse | undefined> {
-    const thisRequestId = this.httpManager.updateRequestId('universalAutoComplete');
+  async executeUniversalAutoComplete(): Promise<AutocompleteResponse> {
     const query = this.state.query.input || '';
-    const results = await this.core.universalAutocomplete({
+    return this.core.universalAutocomplete({
```

Comment:
I'm wondering if we want to return a different response model than the one from Core, however I think this works

Reply:
I think it makes sense to return as is with the updated filterSearchResponse interface in core, which now should only include `sections` and some ids (queryId, businessId, and uuid).